sceptre/plan/plan.py

```diff
@@ -73,10 +73,10 @@ def _generate_launch_order(self, reverse=False) -> List[Set[Stack]]:
                 graph.remove_stack(stack)
 
         if not launch_order:
-            raise ConfigFileNotFoundError(
-                "No stacks detected from the given path '{}'. Valid stack paths are: {}"
-                .format(sceptreise_path(self.context.command_path), self._valid_stack_paths())
-            )
+            error_text = f'No stacks detected from the given path {sceptreise_path(self.context.command_path)}.'
+            if len(self._valid_stack_paths()) < 10:
```

Comment:
Let's use a well-named constant here rather than a magic number.

Reply:
I also think we ought to filter the valid stack paths based upon my suggestion here: https://github.com/Sceptre/sceptre/issues/1268#issuecomment-1329332100

Reply:
Constant set and refactored to filter the valid stack paths.